utils/rpms/daos.spec

```diff
@@ -34,6 +34,7 @@ BuildRequires: libevent-devel
 BuildRequires: libyaml-devel
 BuildRequires: libcmocka-devel
 BuildRequires: readline-devel
+BuildRequires: systemd
```

Comment:
Shouldn't this also be a `Requires:` ?

Reply:
Systemd is part of the base system. Requiring it isn't necessary. However to have systemd unit files in the spec you need a BuildRequires otherwise you don't get the right defines.